app/src/androidTest/java/com/infomaniak/drive/ApiRepositoryTest.kt

```diff
@@ -594,7 +600,7 @@ class ApiRepositoryTest : KDriveTest() {
 
             // Make sure the dropbox has been updated
             with(getDropBox(testFolder)) {
-                assertFalse(data?.emailWhenFinished ?: true, "Email when finished should be false")
+                assertFalse(data?.hasNotification ?: true, "Email when finished should be false")
```

Comment:
Same as previous comment.

Reply:
We are in a migration branch, and the tests will be rewritten on another PR